### PR TITLE
Write a proper README file, add contribution instructions for dumper

### PR DESCRIPTION
**Pull request by paolomorandini**
Initial commit for `README` and dumper's `CONTRIBUTING` pages.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,34 @@
...
+This project contains a collection of tools related to [BigQuery Migration Service](https://cloud.google.com/bigquery/docs/migration-intro).
+
+The current available tools are:
+- Metadata and Log Dumper: can generate an archive containing metadata or logs for the currently supported DBMSs, for more information [read the tool documentation](https://cloud.google.com/bigquery/docs/generate-metadata).
+- Batch SQL Translation Client: command line utility to run a Batch SQL Translation job with support for macro expansion/unexpansion, for more information [read how to submit a translation job using the client](https://cloud.google.com/bigquery/docs/batch-sql-translator#submit_a_translation_job).
+
+## Contributing
+
+Contributing instructions are available, per tool, at the following locations:
+- [Batch SQL Translation Client contribution guide](client/CONTRIBUTING.md)
```
Comment (paolomorandini) [inline README.md:13]: Depends on:
> https://github.com/ajwelch4/dwh-migration-tools/compare/aj-validate-paths...ajwelch4:black-formatter

**Review by bdw-g — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1 +1,34 @@
 # Data Warehouse Migration Tools
+[![License](https://img.shields.io/badge/License-Apache_2.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+
+This project contains a collection of tools related to [BigQuery Migration Service](https://cloud.google.com/bigquery/docs/migration-intro).
+
+The current available tools are:
+- Metadata and Log Dumper: can generate an archive containing metadata or logs for the currently supported DBMSs, for more information [read the tool documentation](https://cloud.google.com/bigquery/docs/generate-metadata).
+- Batch SQL Translation Client: command line utility to run a Batch SQL Translation job with support for macro expansion/unexpansion, for more information [read how to submit a translation job using the client](https://cloud.google.com/bigquery/docs/batch-sql-translator#submit_a_translation_job).
```
[the remark this answers is not in the document]
Comment (genemerrill) [reply README.md:8]: URLs look good to me based on the devsite, but do we want to hold this back until they are all available publicly?

**Review by genemerrill — APPROVED** [review]
lgtm

File: README.md [hunk 3]
```diff
@@ -1 +1,34 @@
...
+
+## Contributing
+
+Contributing instructions are available, per tool, at the following locations:
+- [Batch SQL Translation Client contribution guide](client/CONTRIBUTING.md)
+- [Metadata and Log Dumper](dumper/CONTRIBUTING.md)
+
+## Release process
+The release process is documented [here](http://go/dwh-migration-tools-release).
```
[the remark this answers is not in the document]
Comment (paolomorandini) [reply README.md:17]: I think it's a good way to not expose the underlying resource direct link.

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (paolomorandini) [reply README.md:8]: Now the URLs are available and look like they're functioning.

File: client/CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1 @@
+# Contributing to Batch SQL Translation Client
```
Comment (paolomorandini) [inline client/CONTRIBUTING.md:1]: Added to not have a broken link, will get content in a future PR

**Pull request merged**